hugo/layouts/shortcodes/youtube.html

```diff
@@ -0,0 +1,33 @@
+{{- $id := .Get "id" | default (.Get 0) -}}
+{{- $title := .Get "title" | default "" }}
+{{- $image := .Get "image" | default "" }}
+{{- $url := "https://www.youtube.com/watch?v={{ $id }}" }}
+{{- $contentClass := "video__content" }}
+{{- $lazy := cond (or (eq (.Get "lazy") false) (eq (.Get "lazy") "false")) false true -}}
+
+{{ if not $title }}
+    {{- $contentClass = print $contentClass " video__content--no-title" -}}
+{{ end }}
+
+<div class="video" data-youtube="{{ $id }}">
+    {{ if $image }}
+        <img class="video__image" src="{{ $image }}" alt="{{ $title }}"{{ if $lazy }} loading="lazy"{{ end }}>
+    {{ else }}
+        <img class="video__image" src="https://img.youtube.com/vi/{{ $id }}/maxresdefault.jpg" alt="{{ $title }}"{{ if $lazy }} loading="lazy"{{ end }}>
+    {{ end }}
+
+    <a href="{{ $url }}" target="_blank" class="video__button">
+        <div class="{{ $contentClass }}">
+            {{ if $title }}
+                <h2 class="video__title">{{ $title }}</h2>
```

Comment:
not sure if this should be h2 because usually there's already an h2 in the content above the video. Not even sure if it shoud be an h? could be a p? Not sure what's most logic when you view the doc heading structure

Reply:
There is only an h2 above when someone put it there. Our other projects use an h, but you tell me, h or p?

Reply:
lets keep h2 for now